src/content/doc-surrealql/operators.mdx

```diff
@@ -883,3 +883,134 @@ DEFINE INDEX mt_pts ON pts FIELDS point HNSW DIMENSION 4 DIST EUCLIDEAN EFC 150
 SELECT id FROM pts WHERE point <|10,40|> [2,3,4,5];
 ```
 <br /><br />
+
+## Types of operators, order of operations and binding power
+
+To determine which operator is executed first, a concept called "binding power" is used. Operators with greater binding power will operate directly on their neighbours before those with lower binding power. The following is a list of all operator types from greatest to lowest binding power.
+
+### Unary
```

Comment:
Hey @Dhghomon thanks for adding this in. Can we make the H3 values a table. It'll be much nicer for easy look up as well. 

The examples can remain as is! ::Team::

Reply:
@Ekwuno Good idea! Definitely looks nicer.